bin/spec

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env bash
 set -e
-
+docker run -dit -p 27017:27017 --name=test-mongo mongo
 if [ "$#" == "0" ]; then
   node_modules/o-tools/bin/lint
   cucumber-js --tags ~@todo
 else
   DEBUG='*,-express:*,-body-parser:*,-finalhandler,-rails-delegate,-text-stream-search' cucumber-js --tags ~@todo "$@"
 fi
-
+docker rm -f test-mongo
```

Comment:
How long does it take to start the mongo docker image? Can we keep the mongo image running here, so that the next test run doesn't need to start it again? I'd like to keep the test as fast as possible.